package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -1043,6 +1045,30 @@ dnsmasq_start()
 	config_get address_as_local "$cfg" address_as_local 0
 	config_list_foreach "$cfg" "address" append_address "$address_as_local"
 
+	# --fast-dns-retry=[<initial retry delay in ms>[,<time to continue retries in ms>]]
+	local fast_dns_retry fast_dns_retry_param
+	config_get_bool fast_dns_retry "$cfg" fast_dns_retry 0
+	[ "$fast_dns_retry" = "1" ] && {
+		config_get fast_dns_retry_param "$cfg" fast_dns_retry_param
+		if [ -n "$fast_dns_retry_param" ]; then
+			append_parm "$cfg" fast_dns_retry_param "--fast-dns-retry"
+		else
+			append_bool "$cfg" fast_dns_retry "--fast-dns-retry"
+		fi
+	}
+
+	# --use-stale-cache[=<max TTL excess in s>]
+	local use_stale_cache stale_cache_param
+	config_get_bool use_stale_cache "$cfg" use_stale_cache 0
+	[ "$use_stale_cache" = "1" ] && {
+		config_get stale_cache_param "$cfg" stale_cache_param
+		if [ -n "$stale_cache_param" ]; then
+			append_parm "$cfg" stale_cache_param "--use-stale-cache"
+		else
+			append_bool "$cfg" use_stale_cache "--use-stale-cache"
+		fi
+	}
```

Comment:
This is likely to create more problems, and I don't think we should reveal this in the configs.